foundation-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -1598,6 +1665,7 @@ To access IDFA and handle it correctly in your mobile application, see the [Appl
 {% hint style="warning" %}
 Starting iOS 14+, applications must use the [App Tracking Transparency](https://developer.apple.com/documentation/apptrackingtransparency) framework to request user authorization before using the Identifier for Advertising (IDFA).
 {% endhint %}
+**Swift**
```

Comment:
Let "enter" **Swift**  to the next line just in case.  It might show up right after the hint.